ibm/service/iampolicy/resource_ibm_iam_authorization_policy.go

```diff
@@ -29,21 +29,19 @@ func ResourceIBMIAMAuthorizationPolicy() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"source_service_name": {
-				Type:         schema.TypeString,
```

Comment:
do we need at least one of these `source_resource_group_id` or `source_service_name` ?

Reply:
Hmm. I'm not sure. 
I think source_service_name should be completely optional. It should be one of those (or subject_attributes) when present. But, if specifying "ExactlyOneOf" or "AtLeastOneOf" then it will throw an error if missing. If we leave this optional, then we may be able to avoid terraform code changes in future when we make changes to what can be a valid policy subject in the API.